nonlineargot.py

```diff
@@ -0,0 +1,42 @@
+#!/bin/python3
+# -*- coding: utf-8 -*-
+
+import math
+import os
+
+#maximum number of characters in the string according to hackerrank
+NO_OF_CHARS = 100000
+
+# Complete the gameOfThrones function below.
+def gameOfThrones(s):
+    #create countermatrix full of zeros
+    count = [0]*NO_OF_CHARS
+
+    #count character occurences
+    for i in range(len(s)):
+        count[ord(s[i])] = count[ord(s[i])] + 1
+
+    #count odd occuring characters
+    odd = 0
+    for i in range(NO_OF_CHARS):
+        if count[i]&1:
```

Comment:
A little bit odd way of checking if a number is odd. Why not use the typical ` x % 2` ?